CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Updated `tox` and Travis configs to improve efficiencies, remove deprecated settings, and implement current best practices [#16](https://github.com/azavea/django-ecsmanage/pull/16)
 - Moved most package distribution configuration from setup.py to setup.cfg [#16](https://github.com/azavea/django-ecsmanage/pull/16)
+- Migrate from Travis to GitHub Actions [#22](https://github.com/azavea/django-ecsmanage/pull/22)
```

Comment:
Unrelated to this line, but can you also please update the CI badge on the `README`? Also, can you please disable this project in Travis CI after merge?

Reply:
I didn't see an option to disable the project in https://travis-ci.org/github/azavea/django-ecsmanage, but I did delete the Travis notify webhook for this repository.

Reply:
I believe it exists at an account/organization level: https://travis-ci.org/organizations/azavea/repositories

(I togged it to off.)

Reply:
Thank you! I don't think I have access to that? Says I'm not a member of any organizations.